home/navigation/Navigation.tsx

```diff
@@ -169,116 +167,66 @@ function DiagnosticsStackScreen() {
   );
 }
 
-const BottomTab = createBottomTabNavigator();
-
-const MaterialBottomTab = createMaterialBottomTabNavigator();
-
 const RootStack = createStackNavigator();
 
-export default (props: { theme: string }) => (
-  <NavigationContainer theme={Themes[props.theme]}>
-    <RootStack.Navigator initialRouteName="Tabs" mode="modal">
-      <RootStack.Screen name="Tabs" options={{ headerShown: false }}>
-        {() => {
-          const projectsNavigationOptions = () => ({
-            tabBarIcon: props => <Entypo {...props} style={styles.icon} name="grid" size={24} />,
-            tabBarLabel: 'Projects',
-          });
-          const profileNavigationOptions = () => ({
+function TabNavigator(props: { theme: string }) {
+  const initialRouteName = Environment.IsIOSRestrictedBuild
+    ? 'ProfileStackScreen'
+    : 'ProjectsStack';
+
+  return (
+    <BottomTab.Navigator {...getNavigatorProps(props)} initialRouteName={initialRouteName}>
+      <BottomTab.Screen
+        name="ProjectsStack"
+        component={ProjectsStackScreen}
+        options={{
+          tabBarIcon: props => <Entypo {...props} style={styles.icon} name="grid" size={24} />,
+          tabBarLabel: 'Projects',
+        }}
+      />
+      {!Environment.IsIOSRestrictedBuild && (
+        <BottomTab.Screen
+          name="ExploreStack"
+          component={ExploreStackScreen}
+          options={{
             tabBarIcon: props => (
-              <Ionicons {...props} style={styles.icon} name="ios-person" size={26} />
+              <Ionicons {...props} style={styles.icon} name="ios-search" size={24} />
             ),
-            tabBarLabel: 'Profile',
-          });
-          const diagnosticsNavigationOptions = () => ({
+            tabBarLabel: 'Explore',
+          }}
+        />
+      )}
+      {Platform.OS === 'ios' && (
+        <BottomTab.Screen
+          name="DiagnosticsStack"
+          component={DiagnosticsStackScreen}
+          options={{
             tabBarIcon: props => (
               <Ionicons {...props} style={styles.icon} name="ios-git-branch" size={26} />
             ),
             tabBarLabel: 'Diagnostics',
-          });
-          const exploreNavigationOptions = () => ({
-            tabBarIcon: props => (
-              <Ionicons {...props} style={styles.icon} name="ios-search" size={24} />
-            ),
-            tabBarLabel: 'Explore',
-            tabBarOnPress: ({ navigation, defaultHandler }) => {
-              if (!navigation.isFocused()) {
-                defaultHandler();
-                return;
-              }
-
-              navigation.popToTop();
-
-              if (navigation.state.routes[0].index > 0) {
-                navigation.navigate('Explore');
-              } else {
-                navigation.emit('refocus');
-              }
-            },
-          });
```

Comment:
@brentvatne I removed this onPress logic since it appears to be unused. I tested that tapping the explore tab returns to the explore page on iOS and Android.

Reply:
:+1: this used to be used when search existed but it's gone now